### PR TITLE
chore(admin): reorganized admin sections and menu items

### PR DESCRIPTION
**Pull request by jdalsem**
BREAKING CHANGE:
Admin menu items and some admin menu item views are no longer present or
have been moved to other locations. Update usage accordingly.

Todo:
 - [x] deprecate elgg_register_admin_menu_item as this is a source of confusion
 - [x] move menu item registration to a hook
 - [x] move elgg install info from statistics to server info
 - [x] add docs about which section is meant for what purpose
 - [x] check old admin page links

### CODE REVIEW

**Comment by jdalsem** [comment]
image

Administer:
Daily tasks, User management, Actionable tasks

Configure:
Settings, Plugin settings, Utilities to configure stuff

Information:
Info, statistics, status

File: engine/lib/admin.php [hunk 1]
```diff
@@ -471,6 +353,170 @@ function _elgg_admin_footer_menu($hook, $type, $return, $params) {
...
+	]);
+	$return[] = \ElggMenuItem::factory([
+		'name' => 'server',
+		'href' => 'admin/server',
+		'text' => elgg_echo('admin:server'),
+		'section' => 'information',
+	]);
+		
+			
+	// plugin settings are added in _elgg_admin_add_plugin_settings_menu() via the admin page handler
```
Comment (mrclay) [inline engine/lib/admin.php]: Good comments!

File: languages/en.php [hunk 2]
```diff
@@ -619,35 +619,35 @@
 
 	'admin:options' => 'Admin options',
 
-	'admin:administer_security' => 'Security',
```
Comment (mrclay) [inline languages/en.php]: How could we avoid changing all these language keys? Maybe a little inline function to rewrite keynames?

Comment (mrclay) [inline languages/en.php]: ...not a dealbreaker though.

Comment (jdalsem) [inline languages/en.php]: security stuff was only part of master... i tried to keep the changes to a minimum

**Pull request merged**